praw/models/reddit/mixins/inboxable.py

```diff
@@ -25,6 +50,18 @@ def mark_read(self):
         .. note:: This method pertains only to objects which were retrieved via
                   the inbox.
 
+        Example usage:
+
+        .. code:: python
+
+           inbox = reddit.inbox.unread()
+
+           # mark all unread inbox items as read
+           for message in inbox:
+               message.mark_read()
+
+        See also :meth:`~.mark_unread`
```

Comment:
Can we also add a "See also" for inbox.mark_read?

Reply:
It's already there - [line 82](https://github.com/Theonefoster/praw/blob/a53d7cbb90e09a944bc2c330e1fa5d2f379900f6/praw/models/reddit/mixins/inboxable.py#L82)

Reply:
Oh - I see what you mean. Good idea - I'll add it.

Reply:
I couldn't get this to work. I looked at the sphynx documentation and found the correct usage, then tried every syntax and reference path variation under the sun and I couldn't get the link to generate correctly - Sphynx kept saying "reference target not found" and no link was generated in the docs. 

I could mention the correct method without the clickable link?

Reply:
The following works for me:

    :meth:`praw.models.Inbox.mark_read`

Reply:
Added